tssc/step_implementers/tag_source/git.py

```diff
@@ -201,11 +202,20 @@ def _git_url(self):
                     _encoding='UTF-8',
                     _decode_errors='ignore'
                 )
-                return_val = out.getvalue().rstrip()
+                git_url = out.getvalue().rstrip()
+
+                # remove ANYTHING@ from begining of git_url since step will pass in its own
+                # username and password
+                #
+                # Regex:
```

Comment:
Documentation update - first capture group `(http://|https://)`, second capture group `(.*)`

Reply:
thats just cuz this includes https://github.com/rhtconsulting/tssc-python-package/pull/84 for ease of testing. but rebased this on the fix in https://github.com/rhtconsulting/tssc-python-package/pull/84. but in any case, https://github.com/rhtconsulting/tssc-python-package/pull/84 should be merged first.